Makefile

```
@@ -43,9 +43,18 @@ GOOS=$(shell go env GOOS)
 GOARCH=$(shell go env GOARCH)
 DOCKER_NAMESPACE=amazon
 COMPONENT=awscollector
+
 LINT=$(PWD)/bin/golangci-lint
 STATIC_CHECK=$(PWD)/bin/staticcheck
 
+TOOLS_MOD_DIR := ./tools/linters
+TOOLS_DIR := $(abspath ./.tools/linters)
+
+MULTIMOD = $(TOOLS_MOD_DIR)/multimod
```

```suggestion
MULTIMOD = $(TOOLS_DIR)/multimod
```

This should reference the built binary, right?

---

Yes. It should be reference to the built binary. Changed. Thanks for the call